### PR TITLE
GCE_CR fix for available images

### PR DESCRIPTION
**Pull request by adarshdubey-star**
A enhancement done on the wrapanapi to incorporate Projects while listing images : https://github.com/RedHatQE/wrapanapi/pull/461
Test result:

```
pytest tests/foreman/api/test_computeresource_gce.py -k test_positive_check_available_images
============================================================================ test session starts ============================================================================
platform linux -- Python 3.11.1, pytest-7.2.1, pluggy-1.0.0
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/addubey/work/robottelo, configfile: pyproject.toml
plugins: xdist-3.1.0, services-2.2.1, mock-3.10.0, reportportal-5.1.3, ibutsu-2.2.4, cov-3.0.0
collected 9 items / 8 deselected / 1 selected                                                                                                                               

tests/foreman/api/test_computeresource_gce.py .                                                                                                                       [100%]

=============================================================== 1 passed, 8 deselected, 6 warnings in 47.16s ================================================================
```

### CODE REVIEW

File: tests/foreman/api/test_computeresource_gce.py [hunk 1]
```diff
@@ -204,18 +205,28 @@ def test_positive_crud_gce_cr(self, module_org, module_location, gce_cert):
         assert not entities.GCEComputeResource().search(query={'search': f'name={new_name}'})
 
     @pytest.mark.tier3
-    def test_positive_check_available_images(self, module_gce_compute, googleclient):
-        """Verify all the images from GCE are available to select from
+    def test_positive_check_available_images(self, module_gce_compute, googleclient, gce_cert):
```
Comment (shubhamsg199) [inline tests/foreman/api/test_computeresource_gce.py:208]:
```suggestion
    def test_positive_check_available_images(self, module_gce_compute, googleclient, gce_cert):
```
Is `gce_ cert` needed here?

Comment (adarshdubey-star) [reply tests/foreman/api/test_computeresource_gce.py]: Updated!

**Review by ColeHiggins2 — APPROVED** [review]

**Comment by Gauravtalreja1** [comment]
Moving this PR to draft until we get wrapanapi PR merged

**Comment by adarshdubey-star** [comment]
trigger: test-robottelo
pytest: tests/foreman/api/test_computeresource_gce.py -k test_positive_check_available_images

**Comment by adarshdubey-star** [comment]
PRT is failing because the issue is not fixed/cherrypicked in the stream build works fine with N-minus versions locally.

**Review by Gauravtalreja1 — APPROVED** [review]

**Pull request merged**